src/common/Card/style.tsx

```diff
@@ -0,0 +1,62 @@
+import styled from "styled-components";
+
+export const StyledCard = styled.div`
+  background-color: white;
+  border: 1px solid #dbdbdb;
+  border-radius: 8px;
+  overflow: hidden;
+  @media only screen and (max-width: 1200px) {
+    background-color: white;
+    border: 1px solid #dbdbdb;
+    border-radius: 8px;
+    overflow: hidden;
+    margin: 0.5em;
+    min-width: max-content;
+  }
```

Comment:
```suggestion
  @media only screen and (max-width: 1200px) {
    flex-grow: 0;
    flex-shrink: 0;
    margin: 0.5em;
  }
```

This is a change that makes the cards look okay for mobile (check the other suggested change in common/Set/style.tsx)